Common/LeetCode/RangeSumofBST.cs

```diff
@@ -0,0 +1,98 @@
+using Common.Interfaces;
+using Common.Enums;
+
+namespace Common.LeetCode
+{
+    public class RangeSumofBST : ICodeChallenge
+    {
+        public CodeChallengeSource CodeChallengeSource => CodeChallengeSource.LeetCode;
+
+        public int CodeChallengeNumber => 938;
+
+        public string Execute()
+        {
+            // Arrange
+            var node7 = new TreeNode(7);
+            var node3 = new TreeNode(3);
```

Comment:
I think this should be in a unit test?

Reply:
Noted. Think about restructuring everything in the future.